main/lsp/requests/code_action.cc

```diff
@@ -40,8 +40,12 @@ LSPResult LSPLoop::handleTextDocumentCodeAction(unique_ptr<core::GlobalState> gs
             for (auto &autocorrect : error->autocorrects) {
                 UnorderedMap<string, vector<unique_ptr<TextEdit>>> editsByFile;
                 for (auto &edit : autocorrect.edits) {
+                    auto range = loc2Range(*run.gs, edit.loc);
+                    if (range == nullptr) {
```

Comment:
Nit: This is a small loop; you can just do `if (range) { /*rest of loop body */ }`